controller/src/enhancements/player.rs

```diff
@@ -509,6 +526,31 @@ impl Enhancement for PlayerESP {
                         .build();
                 }
             }
+            if settings.esp_distance {
```

Comment:
May put it in the whole esp info clause. Would make more sense, as the font size & position has already been calculated.